test/model/content_test.dart

```diff
@@ -943,6 +943,387 @@ class ContentExample {
+              '<span class="msupsub">'
+                '<span class="vlist-t vlist-t2">'
+                  '<span class="vlist-r">'
+                    '<span class="vlist" style="height:0.1514em;">'
+                      '<span style="top:-2.55em;margin-left:0em;margin-right:0.05em;">'
+                        '<span class="pstrut" style="height:2.7em;"></span>'
+                        '<span class="sizing reset-size6 size3 mtight">'
+                          '<span class="mord mathnormal mtight">n</span></span></span></span>'
+                    '<span class="vlist-s">​</span></span>'
+                  '<span class="vlist-r">'
+                    '<span class="vlist" style="height:0.15em;"><span></span></span></span></span></span></span></span></span></span></span></p>',
+    [
+      MathBlockNode(
+        texSource: 'x_n',
+        nodes: [
+          KatexSpanNode(
+            styles: KatexSpanStyles(),
+            text: null,
+            nodes: [
+              KatexStrutNode(heightEm: 0.5806, verticalAlignEm: -0.15),
+              KatexSpanNode(
+                styles: KatexSpanStyles(),
+                text: null,
+                nodes: [
+                  KatexSpanNode(
+                    styles: KatexSpanStyles(
+                      fontFamily: 'KaTeX_Math',
+                      fontStyle: KatexSpanFontStyle.italic),
+                    text: 'x',
+                    nodes: null),
+                  KatexSpanNode(
+                    styles: KatexSpanStyles(textAlign: KatexSpanTextAlign.left),
+                    text: null,
+                    nodes: [
+                      KatexVlistNode(
+                        rows: [
+                          KatexVlistRowNode(
+                            verticalOffsetEm: -2.55 + 2.7,
+                            node: KatexSpanNode(
+                              styles: KatexSpanStyles(
+                                marginLeftEm: 0,
+                                marginRightEm: 0.05),
+                              text: null,
+                              nodes: [
+                                KatexSpanNode(
+                                  styles: KatexSpanStyles(fontSizeEm: 0.7),
+                                  text: null,
+                                  nodes: [
+                                    KatexSpanNode(
+                                      styles: KatexSpanStyles(
+                                        fontFamily: 'KaTeX_Math',
+                                        fontStyle: KatexSpanFontStyle.italic),
+                                      text: 'n',
+                                      nodes: null),
+                                  ]),
+                              ])),
+                        ]),
+                    ]),
+                ]),
+            ]),
+        ]),
+    ]);
+
+  static const mathBlockKatexVertical3 = ContentExample(
+    'math block katex vertical 3',
```

Comment:
(And I'm not sure what this example demonstrates that isn't covered by the "vertical 1" example above. So that would be good to make explicit.)